### ISSUE TITLE
How to add execution duration at scenario level in cucumber reports

### ISSUE THREAD

**Issue by AdithyaSriramoju04** [damianszczepanik/cucumber-reporting#1115]
Hi there,

If you look at the HTML report per the screenshot below, the scenario has different steps and contains the execution duration at each step. Similarly, how can I add the total execution duration of that scenario at the scenario level ??

![cucumber snap](https://github.com/damianszczepanik/cucumber-reporting/assets/87465005/9524f52e-8d35-4fd4-b35d-cae9ffd915ec)

![cucumber hrml report](https://github.com/damianszczepanik/cucumber-reporting/assets/87465005/37b92602-36be-4384-a536-0d5037a41fa3)

**Comment by damianszczepanik** [comment]
What framework are you using, configuration and output looks different from https://damianszczepanik.github.io/cucumber-html-reports/overview-features.html

**Comment by AdithyaSriramoju04** [comment]
yes, it's a different one. I'm using WebDriverIO with JavaScript - Cucumber framework. Can you please help me on how to set the execution duration at scenario level in cucumber-html-report?

**Comment by damianszczepanik** [comment]
You already have it https://damianszczepanik.github.io/cucumber-html-reports/report-feature_1_1515379431.html
